### ISSUE TITLE
Problems with Tree Generation?

### ISSUE THREAD

**Issue by xiaoruiDong** [ReactionMechanismGenerator/RMG-database#433]
I am creating a new Retroene reaction family, and I got a failure in Travis Push Test. The failed test is `Kinetics family Retroene: Entry is accessible?`, and more details can be found at https://travis-ci.org/github/ReactionMechanismGenerator/RMG-database/builds/714344607.

I guess the error relates to matching to the wrong node? But I am not sure how does that happen. I used the `GenerateTree` script with the training reaction that can be found in `retroene` branch.

@mjohnson541, Matt if you have time, do you mind taking a look at it? Thanks!

**Comment by mjohnson541** [comment]
I don't think this is an issue with tree generation. These look like issues related to sample molecule generation, which we use to run the accessibility tests. It doesn't seem to be capable of respecting the InRing attribute in these cases.

**Comment by xiaoruiDong** [comment]
> I don't think this is an issue with tree generation. These look like issues related to sample molecule generation, which we use to run the accessibility tests. It doesn't seem to be capable of respecting the InRing attribute in these cases.

Thank you, Matt, for pointing out the issue. I am not quite familiar with this actually, Let's discuss in the developer hour if you are available. Thanks
